model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -56,6 +56,30 @@ data class AnalyzerResult(
         errors.any { it.value.isNotEmpty() }
                 || projects.any { it.scopes.any { it.dependencies.any { it.hasErrors() } } }
     }
+
+    /**
+     * A sorted set containing the [Identifier]s of all packages which are not excluded by the repository configuration.
+     */
+    private val includedPackageIds: SortedSet<Identifier> by lazy {
+        projects.flatMap { project ->
+            if (project.excluded) {
+                emptyList()
+            } else {
+                project.scopes.flatMap { scope ->
+                    if (scope.excluded) {
+                        emptyList<Identifier>()
```

Comment:
Maybe use `sortedSetOf()` (which requires no type then) here to be consistent with the return type of `collectDependencyIds` below?

Reply:
Done.